Dockerfile

```diff
@@ -1,9 +1,13 @@
-FROM golang:1.20 as builder
-WORKDIR /netreap
-COPY . /netreap
+FROM --platform=${BUILDPLATFORM} golang:1.20.5 as builder
+WORKDIR /app
+COPY go.mod go.sum ./
+RUN go mod download
+COPY . ./
 ARG VERSION
-RUN go build -ldflags "-s -w -X 'main.Version=$VERSION'"
-
-FROM gcr.io/distroless/base-debian11
-COPY --from=builder /netreap/netreap /usr/bin/netreap
-ENTRYPOINT ["/usr/bin/netreap"]
+ARG TARGETOS
+ARG TARGETARCH
+RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -ldflags "-s -w -X 'main.Version=$VERSION'" -o /netreap
```

Comment:
You really don't need this if you're using buildx since the build happens in the correct platform image. There probably isn't any harm in leaving it since we're not building for arm/7, which needs special handling